client/components/NaviBar.jsx

```diff
@@ -40,19 +40,19 @@ import NavDropdown from 'react-bootstrap/lib/NavDropdown';
     var accountStuff;
     if (!this.props.loggedIn) {
       accountStuff =
-        <NavDropdown open={this.state.open} onClick={event => this.dropdownToggle(event)} 
-          className="dropdown-toggle" eventKey={3} title="Account" id="basic-nav-dropdown">
-          <MenuItem eventKey={3.1}>
+        <NavDropdown open={this.state.open} onClick={event => this.dropdownToggle(event) } onToggle={() => {}}
```

Comment:
added empty function for toggle?

Reply:
Yea, it is a 'bug' fix. When using a NavDropdown component and passing 'open' to props, it also wants a function passed called onToggle. Without it, an error is thrown. I tried replacing onClick with onToggle. It resolved the error but caused a new problem. It would open the dropdown, but as soon as you would click on the form, it would toggle closed. Meaning it was impossible to sign in. I didn't see options in bootstrap to solve it in the allotted time I had, so I opted for an empty function to keep functionality and get rid of the error. I think it could be fixed if it were a button that toggles a div instead of using a dropdown if that is something you would like to refactor.

Reply:
yup, it makes sense. Yea I remember the console yelling at open props too.

good to go!